test/compile/axis/config.test.ts

```diff
@@ -0,0 +1,20 @@
+import {getAxisConfigs} from '../../../src/compile/axis/config';
+
+describe('getAxisConfigs', () => {
+  it('correctly get axis types for orient signal', () => {
+    const {vgAxisConfig} = getAxisConfigs(
+      'x',
+      'point',
+      {signal: 'a'},
+      {
+        axisTop: {labelBaseline: 'line-bottom', labelPadding: 2},
```

Comment:
Can we change one of these config properties to a signal to make sure objects work okay.

Reply:
not sure what you mean by "objects work okay".

Reply:
b/c we use `stringValue` to signalfy the config.

Reply:
if one of the properties is a signal it will stringify to `"{ 'signal': '...' }"`. Should make sure this works properly.